app/views/candidate_mailer/reinstated_offer.text.erb

```diff
@@ -1,13 +1,13 @@
 Dear <%= @application_form.first_name %>
 
-<%= @course_option.course.provider.name %> has confirmed your deferred offer to study <%= @course_option.course.name_and_code %> in <%= @course_option.course.start_date.to_s(:month_and_year) %>.
+<%= @course_option.course.provider.name %> has confirmed your deferred offer to study <%= @course_option.course.name_and_code %>.
 
-This was deferred from (<%= @application_choice.offer_deferred_at.to_s(:month_and_year) %>).
+The course starts <%= @course_option.course.start_date.to_s(:month_and_year) %>.
```

Comment:
Does this need a preposition?

```suggestion
The course starts in <%= @course_option.course.start_date.to_s(:month_and_year) %>.
```

Reply:
I'm presuming that `.to_s(:month_and_year)` means it's always month and year?

Reply:
Yeah, the `:month_and_year` should always display the month and year, like this for example `February 2022`.

Reply:
ah ok cool - if it's always [month][year] then let's add the preposition